README.md

```diff
@@ -1,13 +1,29 @@
 <h1 align=center><img src=https://github.com/enBloc-org/kindly/assets/114364165/ec1855ea-13b7-4d61-800d-ff1f0c90ff70 height=200/></h1>
-Kindly is an open source web application created by Trafalgar Girls and enBloc, supporting Ukrainian refugees in the United Kingdom. Individuals can contribute items at no cost to provide much-needed support to members of the Ukrainian refugee community.
-<br>
-<br>
+
+Kindly is an open source web application created by Trafalgar Girls and <a href="https://github.com/enBloc-org">enBloc</a> that supports Ukrainian refugees in the United Kingdom. Individuals can contribute items at no cost to provide much-needed support to members of the Ukrainian refugee community.
+
+
+- ### Who is it for?
+
```

Comment:
I think sell and buy are inaccurate here.  So would recommend removing.

Reply:
Kindly is for anyone who wants to share items with the Ukrainian refugee community. Equally, it is for members of the Ukrainian refugee community who want to borrow or receive items